src/apps/frontend/pages/authentication/phone-login/phone-login-form.tsx

```diff
@@ -0,0 +1,88 @@
+import { PhoneNumberFormat, PhoneNumberUtil } from 'google-libphonenumber';
+import React from 'react';
+
+import { FormControl, Input } from '../../../components';
+import { AsyncError } from '../../../types';
+
+import CountryCodeSelect from './country-code-select';
+import usePhoneLoginForm from './phone-login-form.hook';
+
+interface PhoneLoginFormProps {
+  onSuccess: () => void;
+  onError: (error: AsyncError) => void;
+}
+
+const PhoneLoginForm: React.FC<PhoneLoginFormProps> = ({ onError, onSuccess }) => {
+  const { formik, isSendOTPLoading } = usePhoneLoginForm({ onSuccess, onError });
+
+  const setFormikFieldValue = (fieldName: string, data: string) => {
+    formik.setFieldValue(fieldName, data)
+      .then().catch((err) => { onError(err as AsyncError); });
+  };
+
+  const handleChangePhone = ({ target }: React.ChangeEvent<HTMLInputElement>) => {
+    const { value } = target;
+    if (!/[a-z]/gi.test(value)) {
```

Comment:
Why is this needed? Same thing with length 3. We should smply use Google Phone lib to do validation instead of rolling out our own validation